ur_robot_driver/src/ros/hardware_interface.cpp

```diff
@@ -89,6 +90,9 @@ bool HardwareInterface::init(ros::NodeHandle& root_nh, ros::NodeHandle& robot_hw
   // to publish correct frame names for frames reported by the robot directly.
   robot_hw_nh.param<std::string>("tf_prefix", tf_prefix_, "");
 
+  // Optional parameter to add a prefix to the 'wrench' hardware interface (if using more than one arm)
+  robot_hw_nh.getParam("wrench_prefix", wrench_prefix);
```

Comment:
Please use the syntax from the parameter above with a default value if the parameter is optional

Reply:
99825fbbf40f6080f81289a12a0f9e2760f1ec2c